docs/jina-ai-cloud/login.md

```diff
@@ -0,0 +1,207 @@
+# Login & Token Management
+
+To use Jina AI Cloud, you need to log in, either via a GitHub or Google account. This section describes how to log in Jina AI Cloud and manage the personal access token. You can do it via webpage, via CLI or via Python API.
+
+## via Webpage
+
+Visit [https://jina.ai](https://jina.ai) and click on the "login" button.
+
+### Login
+
+```{figure} login-1.png
+```
+
+After log in you can see your name and avatar in the top-right corner. 
+
+```{figure} login-2.png
+```
+
+
+### Token Management
+
+You can follow the GUI to create/delete personal access tokens for your Jina applications.
+
+```{figure} pat.png
+```
+
+To use a token, set it as the environment variable `JINA_AUTH_TOKEN`.
+
+## via CLI
+
+### Login
+
+```shell
+jina auth login
+```
+
+This will open browser automatically and login via 3rd party. Token will be saved locally.
```

Comment:
```suggestion
This will open your browser automatically and login via third party. Your token will be saved locally.
```
What does third party mean here? Third party authentication?